static/css/index.css

```diff
@@ -162,4 +191,140 @@ header {
 
 /* end sectionone */
 
+/* start content styles */
+
+.sectiontwo__container__items,
+.sectionthree__container__posts,
+.sectionfour__container__rows,
+.sectionfive__container__services {
+  flex-direction: column;
+  display: grid;
+  grid-row-gap: 13px;
+  border: 2px solid #ddd;
+  padding: 15px 15px 33px 15px;
+  border-radius: 19px;
+  background-color: #fff;
+  grid-template-rows: 0fr 1fr;
+}
+
+.sectionsix__container__skills {
+  flex-direction: column;
+  display: grid;
+  grid-row-gap: 13px;
+  padding: 15px 15px 33px 15px;
+  grid-template-rows: 0fr 1fr;
+}
+
+.sectionsix {
+  background-color: #fff;
+  border-top-right-radius: 73px;
+}
+
+.sectiontwo__container__items__item > h2,
+.sectionthree__container__posts__post > h2,
+.sectionfour__container__rows__row > h2,
+.sectionfive__container__services__service > h2,
+.sectionsix__container__skills__skill > h2 {
+  margin-bottom: 15px;
+  font-size: 32px;
+}
+
+.accordion > li {
+  border: 0;
+  outline: 0;
+  width: 100%;
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  padding: 16px;
+  font-size: 20px;
+  margin-bottom: 25px;
+  border-bottom: 2px solid #ddd;
+}
+
+.first__ul {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 15px;
+}
+
+.second__ul > li {
+  background-color: #ebebff;
+  padding: 3px 5px;
+  border-radius: 5px;
+  margin-right: 15px;
+  color: #6070ff;
+}
+
+.first__ul > li:nth-child(1) {
+  font-weight: bold;
+}
+
+.sectiontwo__container__items__item > p,
+.sectionthree__container__posts__post > p,
+.sectionfour__container__rows__row > p,
+.sectionfive__container__services__service > p,
+.sectionsix__container__skills__skill > p {
+  margin-bottom: 15px;
+}
+
+.second__ul {
+  display: flex;
+  justify-content: flex-start;
+  margin-bottom: 30px;
+  list-style-type: none;
+  padding: 3px 5px;
+}
+
+.sectiontwo__container__items__item > button,
+.sectionthree__container__posts__post > button,
+.sectionfour__container__rows__row > button,
+.sectionfive__container__services__service > button,
+.sectionsix__container__skills__skill > button.wow {
+  outline: 0;
+  padding: 13px;
+  border-radius: 5px;
+  background-color: #fff;
+  border: 1px solid #396df2;
+  color: #396df2;
+}
```

Comment:
- [ ] Kindly make sure for all the buttons you implement the style also when the button will be `hover-overed`, `clicked` and `disabled`

Reply:
@christianbiring1, if this required change is in line with the design, please provide screenshots to provide more context and help the student understand. The screenshot should be a comparison between what has been implemented and the provided design.